openpype/pipeline/create/context.py

```diff
@@ -1117,11 +1107,20 @@ def host_is_valid(self):
 
     @property
     def host_name(self):
+        if hasattr(self.host, "name"):
+            return self.host.name
```

Comment:
Why was this added? Was this wrong originally?

Reply:
It wasn't wrong, but if registered host is class of `HostBase` it's easier to use its `name` attribute.

Reply:
If it's a special case, let's clarify in the code/docstring yes!

Reply:
Ok, check. To me personally this isn't _"easier"_. It's introducing more code and leaves me wondering why it's a 'special case'. I'd rather have it do one thing than branching between two things.

Reply:
Ideal case is that only `self.host.name` is used all the time. The branching is because Publisher is in experimental tools so both must supported.